### PR TITLE
chor(optimize-dependencies): Updated Optimize dependencies for 3.9.2 …

### PR DESCRIPTION
**Pull request by cigdemilhan**
…version

## What is the purpose of the change

_Briefly describe the change you are making, this helps the reviewer by providing an overview._

## Are there related marketing activities

_Are there any marketing activities related to the related feature or component? If so, please briefly describe them._

## When should this change go live?

_Does this change need to be released before a certain date or milestone? If so, when?_

## PR Checklist

- [ ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [ ] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/reference/dependencies.md [hunk 1]
```diff
@@ -919,24 +919,24 @@ All of these libraries are required for core functionality.
 - [classnames@2.3.2](https://github.com/JedWatson/classnames) (MIT)
 - [clsx@1.1.1](https://lukeed.com) (MIT)
 - [comma-separated-tokens@2.0.2](https://wooorm.com) (MIT)
-- [component-event@0.1.4](https://github.com/component/event) (MIT\*)
-- [component-props@1.1.1](https://github.com/component/props) (MIT\*)
+- [component-event@0.1.4](https://github.com/component/event) (MIT-)
+- [component-props@1.1.1](https://github.com/component/props) (MIT-)
```
Comment (akeller) [inline docs/reference/dependencies.md:923]: Are these - additions intentional?

**Comment by akeller** [comment]
There is a markdown formatting issue where the Optimize dependencies render as mailtos, which I've addressed by adding a space as a work around.

File: docs/reference/dependencies.md (hunk 1 above)
Comment (akeller) [reply docs/reference/dependencies.md:923]: @cigdemilhan nudging this, did you mean to add these dashes?

Comment (cigdemilhan) [reply docs/reference/dependencies.md:923]: @akeller I did not mean to add these dashes, probably it happened when I did changes the wildcards with dash

Comment (akeller) [reply docs/reference/dependencies.md:923]: I'll clean this up before merging 🙂

File: docs/reference/dependencies.md [hunk 2]
```diff
@@ -908,8 +908,8 @@ All of these libraries are required for core functionality.
 - [@types/prop-types@15.7.5](https://github.com/DefinitelyTyped/DefinitelyTyped) (MIT)
 - [@types/unist@2.0.3](https://github.com/DefinitelyTyped/DefinitelyTyped) (MIT)
 - [bail@2.0.1](https://wooorm.com) (MIT)
-- [bpmn-js-disable-collapsed-subprocess@0.1.4](https://github.com/barmac) (MIT)
-- [bpmn-js@9.4.0](https://github.com/nikku) (MIT\*)
+- [bpmn-js-disable-collapsed-subprocess@0.1.5](https://github.com/barmac) (MIT)
+- [bpmn-js@9.4.0](https://github.com/nikku) (MIT-)
```
Comment (akeller) [inline docs/reference/dependencies.md:912]:
```suggestion
- [bpmn-js@9.4.0](https://github.com/nikku) (MIT)
```

**Review by akeller — APPROVED** [review]

File: docs/reference/dependencies.md (hunk 1 above)
Comment (cigdemilhan) [reply docs/reference/dependencies.md]: @akeller thanks :)

**Pull request merged**